libegg/Makefile.am

```diff
@@ -32,6 +32,6 @@ libegg_la_CFLAGS =				\
 
 libegg_la_LIBADD = 	\
 	$(LIBEGG_LIBS)	\
-	-lSM -lICE
```

Comment:
Maybe this is missing when we build mate-submodule in a host package, or not?
Rest looks OK.

Reply:
no answer...
we can fix it later, if it is a problem.

Reply:
This submodule was checked with the build of Engrampa, for this reason I added a link to the change necessary to Engrampa to use this change on the PR comment body.

While the merge request was active, Engrampa was dispensed and the configure.ac file was added to test the isolated build of the submodule. I added the required change to configure.ac to use this change.

Yes, to use this change you need to update projects that use this submodule.